modules/ROOT/pages/install-a-bonita-bpm-cluster.adoc

```diff
@@ -162,11 +162,11 @@ In this case you already have a Bonita Platform running as single node installat
 
 === Update the configuration in database
 
-Some properties of the Bonita Platform needs to be changed, through  xref:bonita-bpm-platform-setup.adoc[Bonita platform setup tool], in order to make your installation work as a cluster node.
+Some properties of the Bonita Platform needs to be changed, through  xref:bonita-platform-setup.adoc[Bonita platform setup tool], in order to make your installation work as a cluster node.
```

Comment:
f the Bonita Platform needs to be changed, => f the Bonita Platform need to be changed,

Reply:
Done